Makefile

```diff
@@ -123,4 +127,7 @@ setupgithook:
 	./hack/precommit-hook.sh setup
 	./hack/commit-msg-hook.sh setup
 
-.PHONY: all check fmt test container-build container-push manifests verify-manifests cluster-up cluster-down cluster-sync cluster-functest cluster-clean pull-ci-changes test-courier setupgithook whitespace-commit
+container-build-must-gather:
```

Comment:
please move this up to the other `container-build-...` targets

Reply:
done

Reply:
done

Reply:
done again

Reply:
done done done

Reply:
done done done